src-java/swmanager-topology/swmanager-storm-topology/src/main/java/org/openkilda/wfm/topology/switchmanager/service/SwitchValidateService.java

```diff
@@ -76,7 +76,7 @@ public SwitchValidateService(
 
     @Override
     public void timeout(@NonNull MessageCookie cookie) throws MessageDispatchException {
-        handle(cookie, SwitchValidateEvent.ERROR, SwitchValidateContext.builder().build());
+        handle(cookie, SwitchValidateEvent.ERROR_RECEIVED, SwitchValidateContext.builder().build());
```

Comment:
Why have you changed this event? What happens when a timeout occurs and the FSM is not in the COLLECT_DATA state?

Reply:
Fixed, thanks.